tools/c7n_azure/c7n_azure/resources/sqlserver.py

```diff
@@ -136,6 +137,77 @@ def __call__(self, i):
         return super(AzureADAdministratorsFilter, self).__call__(i['properties']['administrators'])
 
 
+@SqlServer.filter_registry.register('vulnerability-assessment')
+class VulnerabilityAssessmentFilter(Filter):
+    """
+    Filter sql servers by whether they have recurring vulnerability scans
+    enabled.
+
+    :example:
+
+    Find SQL servers without vulnerability assessments enabled.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: sql-server-no-va
+            resource: azure.sql-server
+            filters:
+              - type: vulnerability-assessment
+                enabled: false
+
+    """
+
+    schema = type_schema(
+        'vulnerability-assessment',
+        required=['type', 'enabled'],
+        **{
+            'enabled': {"type": "boolean"},
+        }
+    )
+
+    log = logging.getLogger('custodian.azure.sqldatabase.VulnerabilityAssessmentFilter')
+
+    def __init__(self, data, manager=None):
+        super(VulnerabilityAssessmentFilter, self).__init__(data, manager)
+        self.enabled = self.data.get('enabled')
+
+    def process(self, resources, event=None):
+        resources, exceptions = ThreadHelper.execute_in_parallel(
+            resources=resources,
+            event=event,
+            execution_method=self._process_resource_set,
+            executor_factory=self.executor_factory,
+            log=log
+        )
+        if exceptions:
+            raise exceptions[0]
+        return resources
+
+    def _process_resource_set(self, resources, event=None):
+        client = self.manager.get_client()
+        result = []
+        for resource in resources:
+            if 'c7n:vulnerability_assessment' not in resource['properties']:
```

Comment:
Is it not available via resource graph?

Reply:
I couldn't get it there and the docs didn't mention it like the others so I prefixed it ... not an exact science I guess.